src/rdb.c

```diff
@@ -3108,16 +3111,16 @@ int rdbLoadRioWithLoadingCtx(rio *rdb, int rdbflags, rdbSaveInfo *rsi, rdbLoadin
         } else if (type == RDB_OPCODE_RESIZEDB) {
             /* RESIZEDB: Hint about the size of the keys in the currently
              * selected data base, in order to avoid useless rehashing. */
-            uint64_t expires_size;
             if ((db_size = rdbLoadLen(rdb,NULL)) == RDB_LENERR)
                 goto eoferr;
             if ((expires_size = rdbLoadLen(rdb,NULL)) == RDB_LENERR)
                 goto eoferr;
-            /* Main dictionary will be resized after reading OPCODE set. Resizing only expires here. */
-            dictExpand(db->expires,expires_size);
             continue; /* Read next opcode. */
         } else if (type == RDB_OPCODE_SLOT_INFO) {
-            uint64_t slot_id, slot_size;
+            if (!server.cluster_enabled) {
+                continue; /* Ignore gracefully. */
```

Comment:
again, same bug as before (which was just resolved the other day and now reintroduced.
you can't skip the reading part.
in order to ignore gracefully, you need to read the data and then not use it.

Reply:
this one is still not resolved.

Reply:
ohh, now it is.